doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1,1224 +1,1411 @@
-\chapter[Vernacular commands]{Vernacular commands\label{Vernacular-commands}
-\label{Other-commands}}
-%HEVEA\cutname{vernacular.html}
+.. include:: ../preamble.rst
+.. include:: ../replaces.rst
 
-\section{Displaying}
+.. _vernacularcommands:
+
+Vernacular commands
+=============================
+
+.. _displaying:
+
+Displaying
+--------------
+
+
+.. _Print:
+
+.. cmd:: Print @qualid.
 
-\subsection[\tt Print {\qualid}.]{\tt Print {\qualid}.\comindex{Print}}
 This command displays on the screen information about the declared or
-defined object referred by {\qualid}.
-
-\begin{ErrMsgs}
-\item {\qualid} \errindex{not a defined object}
-\item \errindex{Universe instance should have length} $n$.
-\item \errindex{This object does not support universe names.}
-\end{ErrMsgs}
-
-\begin{Variants}
-\item {\tt Print Term {\qualid}.}
-\comindex{Print Term}\\ 
-This is a synonym to {\tt Print {\qualid}} when {\qualid} denotes a
-global constant. 
-
-\item {\tt About {\qualid}.}
-\label{About}
-\comindex{About}\\ 
-This displays various information about the object denoted by {\qualid}:
-its kind (module, constant, assumption, inductive,
-constructor, abbreviation, \ldots), long name, type, implicit
-arguments and argument scopes. It does not print the body of
-definitions or proofs.
-
-\item {\tt Print {\qualid}@\{names\}.}\\
```

Comment:
Looks like this was removed. Is it a synchronization problem? Indeed, this is something new from 8.8: https://github.com/coq/coq/commit/280c922fc55b57c430cad721c83650a796a375fd

General remark: I learned that people have done the conversion from the HTML version of the 8.7 manual. Very bad idea. This is likely to have created many synchronization problems. Fortunately, the effort by @maximedenes to save the history should help spotting those. I guess we will have to re-review carefully all Sphinx migration PRs.